pkg/util/traceid.go

```diff
@@ -51,6 +51,28 @@ func TraceIDToHexString(byteID []byte) string {
 	return id
 }
 
+// SpanIDToHexString converts a span ID to its string representation and WITHOUT removing any leading zeros.
+// Assuming everything passed in here will be 64 bit or less
+func SpanIDToHexString(byteID []byte) string {
+	id := hex.EncodeToString(byteID)
+	length := len(id)
```

Comment:
lines 58-72 can be written fairly succinctly using a fmt.Sprintf and a format modifier like `"%016s"`. this will likely be a trade off between simplicity and performance. wdyt?

Reply:
lines 59-64 are still a mystery to me. if the length is > 16 and starts with a 0 then take the 16th right most characters in the string?

i think just the `fmt.Sprintf` should handle what we want w/o any of this extra logic.

Reply:
Yes. I just realized that as well.